### ISSUE TITLE
Incorrect or unclear ElasticSearch version for magento 2.4.3-p1 on system requirements

### ISSUE THREAD

**Issue by chavesfop** [magento/devdocs#9396]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which topic?

https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements.html

### What's wrong with the content?

The content says 2.4.3 needs elasticsearch 7.10, and when I try to build a cloud environment on magento cloud this warning appears on logs:

```
[2022-04-01 15:03:21] WARNING: [2017] The current configuration is not compatible with this version of Magento
    Magento 2.4.3.0-patch1 does not support version "7.10" for service "elasticsearch". Service version should satisfy "~7.5.0 || ~7.6.0 || ~7.7.0 || ~7.9.0" constraint.
```

### What changes do you propose?

Do the proper tests on elasticsearch supported versions and fix the documentation.

**Comment by BaDos** [comment]
@chavesfop do you use ece-tools 2002.1.10?

**Comment by chavesfop** [comment]
> @chavesfop do you use ece-tools 2002.1.10?

Yes, in my composer.lock it uses the version 2002.1.10
```
"name": "magento/ece-tools",
            "version": "2002.1.10",
            "dist": {
                "type": "zip",
                "url": "https://repo.magento.com/archives/magento/ece-tools/magento-ece-tools-2002.1.10.0.zip",
                "shasum": "4ef2fea37851746b11dc012801db8056043b4134"
            },
```

**Comment by BaDos** [comment]
@chavesfop This issue was fixed in 2002.1.10
